pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -10,66 +10,84 @@ relevant_links:
+Each channel on the server has a channel description which briefly describes the topics covered by that channel. If you're not sure where to post, feel free to ask us which channel is appropriate in `#community-meta`.
 
 # General Help Channels
 
-Our general help channels move at a fast pace, and attract a far more diverse spectrum of helpers.
-This is a great choice for a generic Python question, and a good choice if you need an answer as soon as possible.
-It's particularly important to [ask good questions](../asking-good-questions) when asking in these channels, or you risk not getting an answer and having your help channel be claimed by someone else.
+Our general help channels cycle quickly, with the advantage of attracting a more diverse spectrum of helpers, and getting individual focus and attention on your question. These channels are a great choice for generic Python help, but can be used for domain-specific Python help as well.
 
-## How To Claim a Channel
+## How to Claim a Channel
 
-There are always 3 available help channels waiting to be claimed in the **Python Help: Available** category.
+There are always three help channels waiting to be claimed in the **Available Help Channels** category.
 
 ![Available help channels](/static/images/content/help_channels/available_channels.png)
+*The Available Help Channels category is always at the top of the server's channel list.*
 
-In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied** category.
+![Available message](/static/images/content/help_channels/available_message.png)
+*This message indicates that a channel is available.*
 
-If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a new help channel every 15 minutes**. However, if you close your help channel using the `!dormant` command, this cooldown is reset early.
+In order to claim one, simply ask your question in one of the available channels. Be sure to [ask good questions](../asking-good-questions) in order to give yourself the best chances of getting help!
 
-![Channel available message](/static/images/content/help_channels/available_message.png)
-*This message is always posted when a channel becomes available for use.*
+![Channel claimed embed](/static/images/content/help_channels/claimed_channel.png)
+*This messages indicates that you've claimed the channel.*
 
-## Q: For how long is the channel mine?
+At this point you will have the **Help Cooldown** role which will remain on your profile until your help channel is closed (with the `!close` command) or goes dormant due to inactivity. This ensures that users can claim only one help channel at any given time, giving everyone a chance to have their question seen.
```

Comment:
Could say "until you close your help channel" to highlight the ability of closing the channel, even if it's not as accurate.

Reply:
Yes